src/test/java/nl/tudelft/ti2206/group9/gui/EndToEndTest.java

```diff
@@ -78,8 +82,10 @@ public void test() throws IOException {
 
 		mainMenu(0);				// Click start
 		playerDies();				// Player dies
+		sleep(COUNTDOWN);
 		deathPopup(0);				// Click "Try Again"
```

Comment:
In my opinion, the `sleep(COUNTDOWN)` should be right after `mainMenu(0)`. In your code, the player dies before the countdown has ended.

<logic,review